Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine
+COPY src/main/webapp/index.html /
+CMD ["/src/main/webapp/index.html"]
```

Comment:
I'm not sure if this is the right source for the Dockerfile... was very confused by the documentation but it seems to work and it is I guess the origin of the app?